src/test/java/test/Sample.java

```diff
@@ -13,5 +13,9 @@ public void sampleTest1() {
 	public void SampleTest2() {
 		System.out.println("Samppe Test2");
 	}
+	@Test
+	public void xyzTask () {
+		System.out.println("new task");
+	}
```

Comment:
What is purpose of this test please eleaborate

Reply:
this is done